Attribution.txt

```diff
@@ -33,9 +30,6 @@ https://github.com/goburrow/modbus/blob/master/LICENSE
 github.com/goburrow/serial (MIT) https://github.com/goburrow/serial
 https://github.com/goburrow/serial/blob/master/LICENSE
 
-github.com/tbrandon/mbserver (MIT) https://github.com/tbrandon/mbserver
```

Comment:
mbserver is still using in the simulator
https://github.com/edgexfoundry/device-modbus-go/blob/ac842be5e75fcd5e4c012862374abb44191cceb5/simulator/go.mod#L5

Reply:
updated